### PR TITLE
Refs #26781 - datepicker css v2

### PR DESCRIPTION
**Pull request by ezr-ondrej**
See comment: https://github.com/theforeman/foreman/pull/6757#discussion_r284237755
Moves the datepicker down to look bit better while it's placed on bottom of the input.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
@MariaAga Mind to take a look, if this is what you've meant?

**Review by MariaAga — DISMISSED** [review]

**Review by ares — CHANGES_REQUESTED** [review]
I have no idea why, but with this, I'm getting this in js console and the menu does not render
```
react-dom.development.js:18396 Uncaught TypeError: Cannot read property 'replace' of null
    at _createCompounder.js:20
    at LayoutSelectors.js:32
    at Array.forEach (<anonymous>)
    at LayoutSelectors.js:30
    at Array.forEach (<anonymous>)
    at patternflyItems (LayoutSelectors.js:28)
    at LayoutSelectors.js:21
    at index.js:76
    at index.js:36
    at index.js:90
```

100% reproducible, reverting fixes immediately

**Comment by tbrisker** [comment]
@ares I think that should be fixed by https://github.com/theforeman/foreman/commit/bfce9f2d7891d0d816a1e24c02f5db294d0d3895

**Comment by ezr-ondrej** [comment]
let me rebase, than :)

**Comment by ezr-ondrej** [comment]
@ares I have rebased, checked and no error for me :)

File: webpack/assets/javascripts/react_app/components/common/DateTimePicker/date-time-picker.scss [hunk 1]
```diff
@@ -1,6 +1,13 @@
+.bootstrap-datetimepicker-widget.dropdown-menu.top {
+  margin-top: -10px;
+}
+
+.bootstrap-datetimepicker-widget.dropdown-menu.bottom {
+  margin-top: 10px !important;
```
Comment (Ron-Lavi) [inline webpack/assets/javascripts/react_app/components/common/DateTimePicker/date-time-picker.scss:6]: We try not to use `!important` if possible,
maybe a small fix is needed in the css of PF core?

**Review by Ron-Lavi — COMMENTED** [review]
Thanks @ezr-ondrej :)
left an inline comment

File: webpack/assets/javascripts/react_app/components/common/DateTimePicker/date-time-picker.scss (hunk 1 above)
Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/common/DateTimePicker/date-time-picker.scss:6]: I took it from comment of Maria, I have tried without beacause I don't like `!important`, didn't work though. And the code should die in next version, so I just don't want to invest too much of effort to it.

**Review by ares — APPROVED** [review]
Thanks @ezr-ondrej, works fine. Mering.

**Pull request merged**